pkg/virt-handler/vm.go

```diff
@@ -2707,6 +2712,25 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 	return nil
 }
 
+func (d *VirtualMachineController) hotplugSriovInterfaces(vmi *v1.VirtualMachineInstance) error {
+	sriovInterfaces := netsriov.FilterSpecInterfacesByBinding(vmi.Spec.Domain.Devices.Interfaces, v1.InterfaceBindingMethod{SRIOV: &v1.InterfaceSRIOV{}})
+	sriovStatusInterfaces := netsriov.FilterStatusInterfacesBySpecInterfaces(vmi.Status.Interfaces, sriovInterfaces)
+	if len(sriovInterfaces) == len(sriovStatusInterfaces) {
+		return nil
+	}
+
+	client, err := d.getVerifiedLauncherClient(vmi)
+	if err != nil {
+		return err
+	}
+
+	if err := isolation.AdjustQemuProcessMemoryLimits(d.podIsolationDetector, vmi); err != nil {
+		d.recorder.Event(vmi, k8sv1.EventTypeWarning, err.Error(), err.Error())
+	}
+
+	return client.HotplugHostDevices(vmi)
```

Comment:
Please add an info log here, this operation is expensive and worth monitoring.

Reply:
There is an a log message on the caller when an error occurs, you mean an info message that it been called?

Reply:
Done